contracts/TablelandTables.sol

```diff
@@ -117,7 +148,7 @@ contract TablelandTables is
         }
```

Comment:
I noticed the conditional statement on line 146 and I was wondering if you could help me understand it better. Specifically, I'm curious about the scenario in which the controller is set to the 0 address. From reading the code, it seems like when the controller is set to the 0 address, all operations are allowed. Can you confirm if this is the intended behavior or if the zero address should have no permissions?
(This comment isn't actually related to changed node, apparently this logic has been the case since before this pull request, just doing a sanity check.)

Reply:
See the doc here: https://github.com/tablelandnetwork/evm-tableland/blob/main/contracts/ITablelandTables.sol#L103

tl;dr, controller = 0 is a special case that is known by the validator. It's confusing... which is part of the motivation for moving to a unified contract-based access control: EVM-22